lms/envs/common.py

```diff
@@ -1733,6 +1733,9 @@
     'cors_csrf',
 
     'commerce',
+
+    # Bookmarks API
+    'bookmarks_api',
```

Comment:
Thats because the data is stored in `openedx/core/djangoapps/content` and only the REST API is in `lms/djangoapps/course_structure_api`. Lets go with just `bookmarks`.